From 785b37b50578bd9135c917977c496fdbdeb25fd7 Mon Sep 17 00:00:00 2001
From: Andreas Fischer <bantu@phpbb.com>
Date: Thu, 13 Aug 2015 15:12:20 -0400
Subject: [PATCH] Remove include statement from BigInteger TestCase.

This was previously required to load MATH_BIGINTEGER_MODE_* constants, but
these are autoloadable via \phpseclib\Math\BigInteger::MODE_* now and the
include path was removed.
---
 tests/Unit/Math/BigInteger/TestCase.php | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/Unit/Math/BigInteger/TestCase.php b/tests/Unit/Math/BigInteger/TestCase.php
index e417b05..0b236d2 100644
--- a/tests/Unit/Math/BigInteger/TestCase.php
+++ b/tests/Unit/Math/BigInteger/TestCase.php
@@ -9,10 +9,7 @@ abstract class Unit_Math_BigInteger_TestCase extends PhpseclibTestCase
 {
     public static function setUpBeforeClass()
     {
-        include_once 'Math/BigInteger.php';
-
         parent::setUpBeforeClass();
-
         self::reRequireFile('Math/BigInteger.php');
     }
 
-- 
2.4.3

